wdl-lint/src/rules/malformed_lint_directive.rs

```diff
@@ -0,0 +1,119 @@
+//! A lint rule for flagging malformed lint directives.
+use wdl_ast::AstToken;
+use wdl_ast::Comment;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Document;
+use wdl_ast::Span;
+use wdl_ast::SupportedVersion;
+use wdl_ast::VisitReason;
+use wdl_ast::Visitor;
+
+use crate::Rule;
+use crate::Tag;
+use crate::TagSet;
+use crate::util::is_inline_comment;
+
+/// The identifier for the Malformed Lint Directive rule.
+const ID: &str = "MalformedLintDirective";
+/// The accepted lint directives.
+const ACCEPTED_LINT_DIRECTIVES: [&str; 1] = ["except"];
+
+/// Creates an "Inline Lint Directive" diagnostic.
+fn inline_lint_directive(span: Span) -> Diagnostic {
+    Diagnostic::warning("lint directive must be on its own line")
+        .with_rule(ID)
+        .with_label("malformed lint directive", span)
+}
+
+/// Creates an "Invalid Lint Directive" diagnostic.
+fn invalid_lint_directive(span: Span) -> Diagnostic {
+    let accepted_directives = ACCEPTED_LINT_DIRECTIVES.join(", ");
+    Diagnostic::warning("lint directive not recognized")
+        .with_rule(ID)
+        .with_label("lint directive not recognized", span)
+        .with_fix(format!(
+            "use any of the recognized lint directives: [{:#?}]",
+            accepted_directives
+        ))
+}
+
+/// Creates a "Missing Whitespace" diagnostic.
+fn missing_whitespace(span: Span) -> Diagnostic {
+    Diagnostic::warning("missing whitespace before lint directive")
+        .with_rule(ID)
+        .with_label("missing whitespace before lint directive", span)
+}
```

Comment:
Same here with the repeated language. We should try to switch it up.
```suggestion
fn missing_whitespace(span: Span) -> Diagnostic {
    Diagnostic::warning("expected exactly one space before lint directive")
        .with_rule(ID)
        .with_label("missing whitespace", span)
}
```

Reply:
Which actually, this makes me realize there's an unhandled edge case. I think that the comment `#@      except: Foo` would get passed your checks without logging any diagnostics, but it also wouldn't be recognized by the directive logic. That's a potentially silent and confusing error.

Can you update your check to see if the present whitespace is _exactly_ one space?

Reply:
I added an excessive whitespace diagnostic to account for this case now! It's present in the latest commit